### PR TITLE
Correct permissions

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/3159

source https://techcommunity.microsoft.com/t5/exchange-team-blog/a-new-recoverableitems-experience-comes-to-exchange-online/ba-p/1505353

"Admins must manually add the Mailbox Import Export RBAC role to accounts they wish to use the Recover Deleted Items feature with."

and also tested, Discovery Management is not enough to see the recover deleted items option in the EAC and to use the cmdlets, removing also the link to an article that does not mention this.

### CODE REVIEW

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/recover-deleted-messages.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ For instructions, see [Connect to Exchange Online PowerShell](/powershell/exchan
 
 ### Step 2: Search for and recover missing items
 
-You need permissions before you can do this procedure or procedures. To see what permissions you need, see the "In-Place eDiscovery" entry in the [Feature permissions in Exchange Online](../../permissions-exo/feature-permissions.md) article.
+You need _Mailbox Import Export_ RBAC role before you can do this procedure or procedures.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/recover-deleted-messages.md:70]:
```suggestion
You need the _Mailbox Import Export_ RBAC role before you can do this procedure or procedures.
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/recover-deleted-messages.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ To verify that you have successfully searched the messages you want to recover,
 
 ### Step 3: Restore recovered items
 
-You need permissions before you can do this procedure or procedures. To see what permissions you need, see the "In-Place eDiscovery" entry in the [Feature permissions in Exchange Online](../../permissions-exo/feature-permissions.md) article.
+You need _Mailbox Import Export_ RBAC role before you can do this procedure or procedures.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/recover-deleted-messages.md:93]:
```suggestion
You need the _Mailbox Import Export_ RBAC role before you can do this procedure or procedures.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**